### PR TITLE
#133, #48: fix StereotypeFontSize 0 and StereotypeFontColor problem

### PR DESCRIPTION
**Pull request by kirchsth**
Hi,

I update the boundary skinparams, ... the 0 size should be fixed (new size 6) and the stereotypetext is drawn with the backgroundcolor (==remains invisible)

It can be tested via my [extended branch](https://github.com/kirchsth/C4-PlantUML/tree/extended)

```
@startuml
!include https://raw.githubusercontent.com/kirchsth/C4-PlantUML/extended/C4_Container.puml

!$COLOR_A_5 = "#7f3b08"
!$COLOR_A_4 = "#b35806"
!$COLOR_A_3 = "#e08214"
!$COLOR_A_2 = "#fdb863"
!$COLOR_A_1 = "#fee0b6"
!$COLOR_NEUTRAL = "#f7f7f7"
!$COLOR_B_1 = "#d8daeb"
!$COLOR_B_2 = "#b2abd2"
!$COLOR_B_3 = "#8073ac"
!$COLOR_B_4 = "#542788"
!$COLOR_B_5 = "#2d004b"

' master doesn't have the new method names
' UpdateSkinparamsAndLegendEntry("boundary", $bgColor=$COLOR_A_5, $fontColor=$COLOR_NEUTRAL, $borderColor=$COLOR_A_1, $shadowing="true")

' my extended supports the new names
UpdateElementStyle("boundary", $bgColor=$COLOR_A_5, $fontColor=$COLOR_NEUTRAL, $borderColor=$COLOR_A_1, $shadowing="true")

System_Boundary(c1, "Sample System") {
    Container(web_app, "Web Application", "C#, ASP.NET Core 2.1 MVC", "Allows users to compare multiple Twitter timelines")
}

System_Boundary(c2, "Sample System") {
}
@enduml
```

BR Helmut

### CODE REVIEW

**Comment by kirchsth** [comment]
fix "...(2-...)" supports following too 
```
' UpdateElementStyle("boundary", $bgColor=$COLOR_A_5, $fontColor=$COLOR_A_3)
UpdateElementStyle("boundary", $fontColor=$COLOR_A_3)
' UpdateElementStyle("boundary", $bgColor=$COLOR_A_5)

' UpdateElementStyle("container", $bgColor=$COLOR_A_5, $fontColor=$COLOR_A_3)
' UpdateElementStyle("container", $fontColor=$COLOR_A_3)
' UpdateElementStyle("container", $bgColor=$COLOR_A_5)
```

**Review by Potherca — APPROVED** [review]

**Comment by Potherca** [comment]
Looks good!

**Comment by cbchenoweth** [comment]
Just pinging back for status update on this one.  Looks like we need another approval from someone with "write access"?  Who is that group / is there a way to ping them on this one?

**Review by adrianvlupu — APPROVED** [review]

**Pull request merged**
